.github/workflows/actions.yml

```diff
@@ -523,6 +523,9 @@ jobs:
         if: matrix.vcpkg-triplet
         shell: bash
         run: |
+          cd $VCPKG_INSTALLATION_ROOT
+          git fetch origin 80f8da23a88a982cf2d81d1a01623c55bb58df15
+          git checkout 80f8da23a88a982cf2d81d1a01623c55bb58df15 -- ports/readline*
```

Comment:
Just to check, where did this commit come from? Should we be reverting to a release tag instead?

Reply:
This commit is from the previous update to the readline-win32 port: https://github.com/microsoft/vcpkg/commits/master/ports/readline-win32/portfile.cmake (PR https://github.com/microsoft/vcpkg/pull/33247)
I guess we could revert to a previous release if you'd like, though vcpkg is updated in gha runners more frequently than releases. Either way for me.
One thing to consider: the [fix for the readline issue](https://github.com/microsoft/vcpkg/pull/34502) has been _just_ pulled in upstream. So maybe we just wait until it makes its way to gha? (a week or two?)